app/main.py

```diff
@@ -1,6 +1,22 @@
+import os
+import requests
+
+URL = "http://api.weatherapi.com/v1/current.json?"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    request = requests.get(URL + f"key={api_key}&q={FILTERING}").json()
+
+    if request.status_code == 200:
+        temp = request["current"]["temp_c"]
+        cond = request["current"]["condition"]["text"]
+
+        print(f"{FILTERING}, weather: {temp} Celsius, {cond}")
+
+    else:
+        return None
```

Comment:
```suggestion
        print("some message")
```
the user must understand if something goes wrong